com.unity.render-pipelines.high-definition/Runtime/Sky/HDRISky/HDRISky.shader

```diff
@@ -122,7 +126,7 @@ Shader "Hidden/HDRP/Sky/HDRISky"
     {
         const float alpha = (_GroundLevel - _WorldSpaceCameraPos.y)/dir.y;
 
-        return _WorldSpaceCameraPos + alpha*dir;
+        return (float3)_WorldSpaceCameraPos + alpha*dir;
```

Comment:
(float3)_WorldSpaceCameraPos  isn't required anymore (see latest staging), please remove it at all location